tests/e2e/tests/admin/logout.spec.ts

```diff
@@ -11,8 +11,8 @@ test.describe('Log Out', () => {
   });
 
   test('a user should be able to logout', async ({ page }) => {
-    await page.getByText('test testing').click();
-    await page.getByText('Logout').click();
+    await page.getByRole('button', { name: 'test testing' }).click();
+    await page.getByRole('button', { name: 'Logout' }).click();
```

Comment:
hey @lucasboilly to pass the e2e test you need to change this line
`await page.getByRole('button', { name: 'Logout' }).click();`
with this one
`await page.getByText(/logout/i).click();`